### DISCUSSION TITLE
NiceGUI Blog post

### DISCUSSION THREAD

**Discussion by CVxTz** [zauberzeug/nicegui#2912] [Show and tell]
Hello ! 

I wrote a blog post about NiceGUI on medium: https://medium.com/p/fb69f14bb0ac 

What do you think about updating the Wiki to add instructions about how to deploy a NiceGUI app on Cloud Run ? I tested it for the demo app of the post and it seems to be holding up fine from the visits of the blog readers.

Thanks!

**Comment by rodja** [comment]
Thank you for this great Medium post. We really appreciate your words. Feel free to add a Cloud Run tutorial to the wiki!

**Reply by CVxTz** [reply]
@rodja I just did. I followed the same structure as the fly.io tutorial.

**Reply by rodja** [reply]
Very nice! Thank you so much.
